### PR TITLE
Lua strings char()

### PR DESCRIPTION
**Pull request by McDuckson**
### Type of Change

- Adding a new entry
- Editing an existing entry (fixing a typo, bug, issues, etc)
- Updating the documentation

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Comment by McDuckson** [comment]
Please tell me what should be changed, thank you

File: content/lua/concepts/strings/terms/char/char.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+**Output**
+
+```codebyte/js
+a
+z
+}
+```
+
+2. The string.char() function can take multiple arguments as well.
```
Comment (SSwiniarski) [inline content/lua/concepts/strings/terms/char/char.md:41]:
```suggestion
## Example 2

The `string.char()` function can take multiple arguments as well:
```

**Comment by McDuckson** [comment]
@SSwiniarski updated file according your comments🤝

**Review by SSwiniarski — APPROVED** [review]
@McDuckson I made some minor edits. Now it looks good for a second review.

**Review by caupolicandiaz — APPROVED** [review]
@McDuckson @SSwiniarski Looks good. Just corrected a typo and made some spacing edits. This is ready to merge

**Pull request merged**